Description: Fix FTBFS with sbuild
 When building with sbuild, all paths contain "/build/", so this
 assertion is always true and files get copied to wrong locations.
Author: Tobias Hansen <tobias.han@gmx.de>

--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -72,7 +72,7 @@
     string(REPLACE "._tx" "" basename ${basename})
 
     set(page ${CMAKE_CURRENT_SOURCE_DIR}/${page})
-    if(${page} MATCHES "/build/")
+    if(${page} MATCHES "/src/build/")
         set(txt_out ${CMAKE_CURRENT_BINARY_DIR}/build/${basename}.txt)
         set(html_out ${CMAKE_CURRENT_BINARY_DIR}/build/${basename}.html)
     else()
